src/util/pelz_request_handler.c

```diff
@@ -2,68 +2,50 @@
 #include "pelz_request_handler.h"
 #include "pelz_log.h"
 #include "common_table.h"
-#include "key_table.h"
 #include "aes_keywrap_3394nopad.h"
 
 #include "sgx_trts.h"
 #include "pelz_enclave_t.h"
 
 RequestResponseStatus pelz_request_handler(RequestType request_type, charbuf key_id, charbuf data, charbuf * output)
 {
-  charbuf key;
   int index;
 
   if (table_lookup(KEY, key_id, &index))
```

Comment:
Does ```table_lookup``` attempt to add the key if it's not already in the table?

Reply:
No, ```table_lookup``` just returns the index of the ```id``` and a ```OK``` status if ```id``` is in the table but returns a ```NO_MATCH``` status if not in the table.